src/main/java/io/cdap/plugin/sfmc/source/MarketingCloudSource.java

```diff
@@ -99,6 +99,7 @@ public void prepareRun(BatchSourceContext context) throws Exception {
     SourceQueryMode mode = conf.getQueryMode(collector);
 
     Configuration hConf = new Configuration();
+    hConf.set("isPreviewEnabled", String.valueOf(context.isPreviewEnabled()));
```

Comment:
make it like "marketing.cloud.preview" and put it in any constant file and use it with constant name.

Reply:
updated